components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1737,6 +1737,154 @@ public static NativeObject jsFunction_getAllPaginatedPublishedAPIs(Context cx, S
 
     }
 
+    /**
+     * This method helps to get all published APIs in paginated form  .
+     *
+     * @param cx      will be used to store information about the executing of the script.
+     *                This is a object of org.mozilla.javascript.Context package.
+     * @param thisObj Object of Scriptable interface provides for the management of
+     *                properties and for performing conversions.
+     * @param args    this will contain parameter list from jag files.
+     * @param funObj  this object  provides for calling functions and constructors.
+     * @return this will return a method which gets list of APIs by status.
+     * @throws ScriptException
+     * @throws APIManagementException
+     */
+    public static NativeObject jsFunction_getAllPaginatedPublishedLightWeightAPIs(Context cx, Scriptable thisObj,
+                                                                                  Object[] args, Function funObj)
+            throws ScriptException, APIManagementException {
```

Comment:
Shall we move this to single line -  `Object[] args, Function funObj) throws ScriptException, APIManagementException {`

Reply:
Thats how its formated through the code formatter.